torch_audiomentations/utils/dsp.py

```diff
@@ -0,0 +1,32 @@
+import librosa
+import numpy as np
+import torch
+
+
+def calculate_rms(samples):
+    """
+    Calculates the root mean square.
+
+                Taken from https://github.com/iver56/audiomentations/blob/master/audiomentations/core/utils.py
+    """
+    return torch.sqrt(torch.mean(torch.square(samples), -1, keepdims=True))
```

Comment:
```suggestion
    return torch.sqrt(torch.mean(torch.square(samples), -1, keepdim=True))
```

ref https://pytorch.org/docs/stable/generated/torch.mean.html